src/componentLibrary/Stack/Stack.tsx

```diff
@@ -0,0 +1,18 @@
+import { StackContainer } from './Stack.style';
+
+interface StackProps {
+  children: React.ReactNode;
+  className?: string;
+  spacing?: 'x1' | 'x2';
```

Comment:
I would probably change this type `x1` and `x2` to just `1` and `2` since plain numbers are used in Grid component prop types. Not a strong opinion tho. I also think once you add the prop for gap to Grid component, its type should match the type used here, maybe also the prop name could be the same, what do you think? :)